packages/connect-form/src/connect-form.tsx

```diff
@@ -15,19 +14,33 @@ import {
 
 import ConnectionStringInput from './connection-string-input';
 
-const logoStyles = css({
-  position: 'absolute',
-  top: spacing[5],
-  left: spacing[5],
+const formContainerStyles = css({
+  margin: 0,
+  marginTop: 0,
+  padding: spacing[4],
+  paddingRight: 0,
+  paddingTop: spacing[5],
+  height: 'fit-content',
+  flexGrow: 1,
+  minWidth: 360,
+  maxWidth: 760,
+  position: 'relative',
+  display: 'inline-block',
 });
 
 const formCardStyles = css({
-  margin: spacing[4],
-  marginTop: 99,
+  margin: 0,
+  marginTop: spacing[6],
+  padding: spacing[2],
   height: 'fit-content',
   width: '100%',
-  minWidth: 360,
-  maxWidth: 800,
+  position: 'relative',
+});
+
+const logoStyles = css({
+  position: 'absolute',
+  top: spacing[5],
+  left: spacing[5],
 });
```

Comment:
While we are refactoring all this stuff, do we want to maybe to something about how the layout is defined here? It seems a bit cumbersome that we have the general layout defined in three different places here where it probably should be in one (which would also allow us to avoid weird absolute styles)

Reply:
Good call, I'll update the layout though to make it flow more (and less absolutes). This logo and a number of the styles added here are very much subject to change pending discussions with design.

Reply:
@gribnoysup  Updated 👍  Made the overflows a bit nicer and added a width breakpoint for the main view so it wraps vertical.